usr/lib/common/trace.c

```diff
@@ -217,6 +222,7 @@ void ock_traceit(trace_level_t level, const char *file, int line,
     char buf[1024];
```

Comment:
I assume buf is still large enough so that the longest trace message plus the tid value still fits?

Reply:
Well, I think the longest trace message from ock I saw was in the range of about 100 characters.
As long as snprintf is used to avoid writing beyond the buffer space I don't really care.